doc/source/release-notes/Percona-Server-5.6.40-84.0.rst

```diff
@@ -0,0 +1,156 @@
+.. rn:: 5.6.40-84.0
+
+============================
+|Percona Server| 5.6.40-84.0
+============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.40-84.0 on
+May 30, 2018 (Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.40-84.0/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.40
+<http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-40.html>`_, including
+all the bug fixes in it, |Percona Server| 5.6.40-84.0 is the current GA release
+in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source
+and free, all the details of the release can be found in the `5.6.40-84.0
+milestone at Launchpad
+<https://launchpad.net/percona-server/+milestone/5.6.40-84.0>`_.
```

Comment:
This is a dead link, because we don't use Launchpad anymore.  This should instead link into public JIRA, probably to the release ticket.